pkgs/development/libraries/mlt/qt-5.nix

```diff
@@ -55,7 +58,11 @@ mkDerivation rec {
     ladspa-sdk
     ladspaPlugins
     rubberband
-  ];
+  ] ++ lib.optionals enableGlaxnimate ([
+    # Enable glaxnimate https://github.com/NixOS/nixpkgs/issues/209923;
+    glaxnimate
+    # Not sure why, but mlt needs to compile itself glaxinate, instead of picking the parent project (that, admitingly, only contains the binary)
```

Comment:
Maybe it's possible to build glaxnimate in a different way such that it produces a library? Maybe the glaxnimate derivation can be changed to contain several outputs?

Reply:
Could be, but I found no easy way reading at the documentation/cmake file, and I'm not extremely fluent with cmake files nor the internal details of these prolects.

Reply:
I have next to no idea about cmake, or C/C++ building in general :/ I think we should rather stall this PR here until there is some feedback from the mlt folks, or another idea by someone else from nixpkgs.

Reply:
if glaxnimate always requires it's buildInputs to be used as a library we should propagate them instead.

Reply:
Well I use `glaxnimate.buildInputs` only because `mlt` tries to compile glaxnimate from scratch, and I guess it's cleaner to bring all dependencies this way rather than copy pasting the 10 dependencies of glaxnimate.

Reply:
@turion so I got some feedbacks from MLT: they won't do anything themself but are open to PR. Regarding glaxnimate, they want to refactor the project to provide explicit small reusable libraries, but so far they had no time to do it. So for now I don't think they want to change their build system to include libraries that won't be available anymore after the refactorization.